### ISSUE TITLE
 Tables/Grid/DataTable Accessibilty Support

### ISSUE THREAD

**Issue by aradhikaIbm** [IBM/carbon-components-angular#53]
Tables/Grid/DataTable Accessibility Support

## Detailed description

> Describe in detail the issue you're having.
We are trying to meet accessibility guidelines in regards to the keyboard navigation on data tables/grids. 
Per the guidelines, https://www.w3.org/TR/wai-aria-practices/#grid, we should be able to focus on individual cells. We are hoping to use this widget library to help meet those requirements. What is outlook for getting keyboard support for the table or adding a grid widget ?

> Is this a feature request (new component, new icon), a bug, or a general issue?
General accessibility issue with tables. 

> Is this issue related to a specific component?
Tables

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
IBM Watson Recruitment, Accessibility deadline of Dec 10, 2018.

**Comment by alwilkin** [comment]
Is there any ETA on when this enhancement will be delivered? And what is the latest version of the angular carbon component from https://github.com/IBM/carbon-components-angular/

**Comment by youda97** [comment]
## DAP Analysis
### default

Level | Description
-- | --
V | Each form control must have associated label. | Details
V | Provide descriptive text in label elements. | Detailss
V | Minimum contrast of 1.00 with its background is not sufficient to meet WCAG AA requirements for text of size 14px and weight of 700. | Details
V | All content must reside within a WAI-ARIA landmark or labelled region role.

**Comment by zvonimirfras** [comment]
Carbon CSS, Vanilla and React currently don't support grid pattern and those interactions are not required for the table component.

Grid pattern is nice to have and we'll get on it as soon as design agrees on what that will look like.

**Comment by zvonimirfras** [comment]
Some DAP issues are related to the "label-less" checkbox and cannot reasonable be handled here, but will be addressed with https://github.com/IBM/carbon-components/issues/964
